### PR TITLE
Add audit logging to createSubjectAccessReportPost

### PR DESCRIPTION
**Pull request by CslBing**
Add audit logging to createSubjectAccessReportPost

### CODE REVIEW

**Review by HorstmannA — APPROVED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -59,14 +59,30 @@ services:
       - "8081:8080"
     healthcheck:
       test: ["CMD", "curl", "-f", "http://localhost:8080/health/ping"]
-    depends_on:
-      hmpps-auth:
-        condition: service_healthy
```
Comment (HorstmannA) [inline docker-compose.yml]: Just checking, is this not needed?

File: docker-compose.yml [hunk 2]
```diff
@@ -11,10 +62,6 @@ services:
     environment:
       - SERVER_PORT=8080
       - SPRING_PROFILES_ACTIVE=dev
-      - DATABASE_URL=db:5432
-      - DATABASE_NAME=postgres
-      - DATABASE_USERNAME=postgres
-      - DATABASE_PASSWORD=postgres
```
Comment (HorstmannA) [inline docker-compose.yml:17]: Why have these been removed?

File: docker-compose.yml (hunk 1 above)
Comment (CslBing) [reply docker-compose.yml]: Those are to make sure the auth service are up running correctly before start service

File: docker-compose.yml (hunk 2 above)
Comment (CslBing) [reply docker-compose.yml:17]: Merge issue added back

**Review by HorstmannA — APPROVED** [review]

**Pull request merged**